<launch>
	<arg name="suffix" default="" />
	<arg name="marker_size" default="5.7" />
	<arg name="max_new_marker_error" default="0.11" />
	<arg name="max_track_error" default="0.5" />

	<arg name="cam_image_topic" default="/camera/depth_registered/points" />
	<arg name="cam_info_topic" default="/camera/rgb/camera_info" />		
	<arg name="output_frame" default="/base_link" />
	<!-- <arg name="output_frame" default="/global_map" /> -->

	<node name="ar_track_alvar$(arg suffix)" pkg="ar_track_alvar" type="individualMarkers" respawn="false" output="screen" args="$(arg marker_size) $(arg max_new_marker_error) $(arg max_track_error) $(arg cam_image_topic) $(arg cam_info_topic) $(arg output_frame)" />
</launch>
